README.md

```diff
@@ -1,92 +1,210 @@
+
+```sh
+cd ~
+git clone https://github.com/openstack-k8s-operators/install_yamls.git
+git clone https://github.com/openstack-k8s-operators/glance-operator.git
+```
+
+### OpenShift cluster
+
+There are many ways get an OpenShift cluster, and our recommendation for the
+time being is to use [OpenShift Local](https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.5/html/getting_started_guide/index)
+(formerly known as CRC / Code Ready Containers).
+
+To help with the deployment we have [companion development tools](https://github.com/openstack-k8s-operators/install_yamls/blob/master/devsetup)
+available that will install OpenShift Local for you and will also help with
+later steps.
+
+Running OpenShift requires a considerable amount of resources, even more when
+running all the operators and services required for an OpenStack deployment,
+so make sure that you have enough resources in the machine to run everything.
+
+You will need at least 5 CPUS and 16GB of RAM, preferably more, just for the
+local OpenShift VM.
+
+**You will also need to get your [pull-secrets from Red Hat](
+https://cloud.redhat.com/openshift/create/local) and store it in the machine,
+for example on your home directory as `pull-secret`.**
+
+```sh
+cd ~/install_yamls/devsetup
+PULL_SECRET=~/pull-secret CPUS=6 MEMORY=20480 make download_tools crc
+```
+
+This will take a while, but once it has completed you'll have an OpenShift
+cluster ready.
+
+Now you need to set the right environmental variables for the OCP cluster, and
+you may want to logging to the cluster manually (although the previous step
+already logs in at the end):
+
+```sh
+eval $(crc oc-env)
+```
+
+**NOTE**: When CRC finishes the deployment the `oc` client is logged in, but
+the token will eventually expire, in that case we can login again with
+`oc login -u kubeadmin -p 12345678 https://api.crc.testing:6443`.
+
+Let's now get the cluster version confirming we have access to it:
+
+```sh
+oc get clusterversion
 ```
-# clone the install_yamls repository
-git clone https://github.com/openstack-k8s-operators/install_yamls
 
-# move to the install_yamls directory
-cd install_yamls
+If you are running OCP on a different machine you'll need additional steps to
+[access its dashboard from an external system](https://github.com/openstack-k8s-operators/install_yamls/tree/master/devsetup#access-ocp-from-external-systems).
 
-# one time operation to initialize PVs within the CRC VM, required by glance to start
+### Storage
+
+There are 2 kinds of storage we'll need: One for the pods to run, for example
+for the MariaDB database files, and another for the OpenStack services to use
+for the VMs.
+
+To create the pod storage we run:
+
+```sh
+cd ~/install_yamls
 make crc_storage
+```
+
+As for the storage for the OpenStack services, at the time of this writing only
+NFS and Ceph are supported. The [Glance Spec](#example-configure-glance-with-ceph-backend)
```

Comment:
If we intend `File` for `NFS` I guess we should be more specific here, otherwise I'm fine just saying that File and Ceph are supported.